server/routes/calendar.js

```diff
@@ -5,6 +5,8 @@ var pool = require("../db/db");
 
 router.post("/save", (req, res) => {
```

Comment:
save doesn't save to the correct calendar - it looks like it's only updating the person with userId 1. To reproduce go to /calendar/2, make a change, save the changes, and refresh.